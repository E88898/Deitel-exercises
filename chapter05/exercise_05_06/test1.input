88
65
2
77
43
90
45
76
9999

