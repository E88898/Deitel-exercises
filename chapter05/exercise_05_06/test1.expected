
The average is: 60.75
